config.json

```
@@ -666,7 +666,38 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "title": "Reference Counting",
```

Maybe change this to:

```suggestion
      "title": "Automatic Memory Management",
```